sonaranalyzer-dotnet/src/SonarAnalyzer.Vsix/SonarAnalyzer.Vsix.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
-    <MinimumVisualStudioVersion>14.0</MinimumVisualStudioVersion>
+    <MinimumVisualStudioVersion>15.0</MinimumVisualStudioVersion>

```

Comment:
Because it is the only project with a strong dependency on the Visual Studio version (because of the vsix format change).